utils/ssh.py

```diff
@@ -15,7 +15,8 @@
 from utils.path import project_path
 from utils.timeutil import parsetime
 
-
+from utils.path import log_path
```

Comment:
utils.path is already imported above, these imports should be combined (`from utils.path import log_path, project_path`)